packages/devtools/src/composite.ts

```diff
@@ -116,9 +181,10 @@ async function loadModelsFromCommits<Models = Record<string, StreamCommits>>(
         genesis,
         MODEL_GENESIS_OPTS
       )
-      assertSupportedModelController(model)
       for (const commit of updates) {
-        await ceramic.applyCommit(model.id, commit as unknown as SignedCommit)
+        const signedCommitContainer = commit as unknown as SignedCommitContainer
+        assertSupportedReadModelController(model, signedCommitContainer)
```

Comment:
Shouldn't we only do this check if it's a signed container? AnchorCommits won't be signed.

Reply:
We also need to check if `genesis` with `assertSupportedReadModelController`.

Reply:
> Shouldn't we only do this check if it's a signed container? AnchorCommits won't be signed.

Do we need to validate AnchorCommit here? possible to do it without network request?

Reply:
> Shouldn't we only do this check if it's a signed container? AnchorCommits won't be signed.
I was assuming they are all signed commit from the function signature. How do I handle the anchor commit here ?
```typescript
async function loadModelsFromCommits<Models = Record<string, StreamCommits>>(
  ceramic: CeramicApi,
  modelsCommits: Models & {} // eslint-disable-line @typescript-eslint/ban-types
): Promise<Record<keyof Models, ModelDefinition>> {
//...
export type StreamCommits = Array<DagJWSResult>;
//...
export declare type SignedCommitContainer = DagJWSResult;

```

Reply:
> We also need to check if `genesis` with `assertSupportedReadModelController`.

How about `static async fromModels(params: FromModelsParams): Promise<Composite> ` Do we also need to validate AnchorCommit and genesis commit? Does `loadStreamCommits` only return genesis and signed commits?

Reply:
`loadStreamCommits()` loads all the commits, including anchor ones, but they are filters to only signed commits in the following logic.
Assuming there is no expiration date on the CACAO I don't think we need to care about the anchor commits, unless I'm missing something? cc @oed

Reply:
We don't need to validate the anchor commits.